packages/core/contracts/financial-templates/long-short-pair/LongShortPair.sol

```diff
@@ -220,17 +212,17 @@ contract LongShortPair is Testable, Lockable {
             expiryPrice = _getOraclePriceExpiration(expirationTimestamp);
             // Cap the return value at 1.
             expiryPercentLong = Math.min(
-                financialProductLibrary.computeExpiryTokensForCollateral(expiryPrice),
+                financialProductLibrary.percentageLongCollateralAtExpiry(expiryPrice),
                 FixedPoint.fromUnscaledUint(1).rawValue
             );
             contractState = ContractState.ExpiredPriceReceived;
         }
 
-        longToken.burnFrom(msg.sender, longTokensToRedeem);
-        shortToken.burnFrom(msg.sender, shortTokensToRedeem);
+        require(longToken.burnFrom(msg.sender, longTokensToRedeem));
+        require(shortToken.burnFrom(msg.sender, shortTokensToRedeem));
 
-        // expiryPercentLong is a number between 0 and 1e18. 0 means all collateral goes to short tokens and
-        // 1 means all collateral goes to the long token. Total collateral returned is the sum of payouts.
```

Comment:
OZ comment on misleading comments:
- The code base is inconsistent about using `1e18` to represent `100%`. For example, the [`expiryPercentLong` comment](https://github.com/UMAprotocol/protocol/blob/b508f536ddfd94a79f93f633142bdc868b73461c/packages/core/contracts/financial-templates/long-short-pair/LongShortPair.sol#L50-L51) switches between `1e18` and `1` in the same comment. This is [repeated later in the contract](https://github.com/UMAprotocol/protocol/blob/b508f536ddfd94a79f93f633142bdc868b73461c/packages/core/contracts/financial-templates/long-short-pair/LongShortPair.sol#L224-L225). Similarly, the `computeExpiryTokensForCollateral` functions in all the libraries [claim the function returns a number between 0 and 1](https://github.com/UMAprotocol/protocol/blob/b508f536ddfd94a79f93f633142bdc868b73461c/packages/core/contracts/financial-templates/common/financial-product-libraries/long-short-pair-libraries/BinaryOptionLongShortPairFinancialProductLibrary.sol#L49), instead of between 0 and 1e18.